SerialPrograms/Source/PokemonSV/Programs/ItemPrinter/PokemonSV_ItemPrinterRNG.cpp

```diff
@@ -207,6 +211,44 @@ void ItemPrinterRNG::value_changed(void* object){
         AFTER_ITEM_PRINTER_DONE_EXPLANATION.set_text("Then stop the program.");
         MATERIAL_FARMER_DISABLED_EXPLANATION.set_text("<br>To enable the Material Farmer, enable \"Automatic Material Farming\" above");
     }
+
+    if (overlapping_bonus()){
+        OVERLAPPING_BONUS_WARNING.set_text(

```

Comment:
Instead of setting and clearing the text, what about showing and hiding the entire option. This will eliminate the extra newline that would be there with an empty message.